src/components/Table/Table.tsx

```diff
@@ -100,7 +91,7 @@ const Thead = ({
   headers,
   isSelectable,
   onSelectAll,
-  showActionsHeader,
+  actionsCount,
```

Comment:
do we really need to pass this in? can't we calculate it in any way?

Reply:
We are using this to determine the size of the Actions th
We can directly pass the size too
I thought this way we can be check for the items instead of if size is present show logic

Reply:
I thought more about being able to grab the list of actions and asking `.length`